mgmtsystem_nonconformity/readme/USAGE.md

```diff
@@ -0,0 +1,48 @@
+To use this module:
+
+- Go to Management System \> Nonconformities
+- Click on Create to enter the following information:
+- Partner : Customer, supplier or internal personnel
+- Related to: Any reference pointing to the NC (order id, project id,
+  task id, etc.)
+- Responsible: Person responsible for the NC
+- Manager : Person managing the department or owner of the procedure
+- Filled in by: Originator of NC report
+- Origins: The source of the NC, how was it discover
+- Procedures: Against which procedure is the NC
+- Description: Evidence, reference to the standards
+- Click on Save and then on Analysis.
+
+Go to the newly created NC and fill in the following information in the
+tab named Causes and Analysis:
+
+- Causes: Add root causes
+- Analysis: Describe the results of the investigation
+- Severity: Select the severity among unfounded, minor and major
+- Immediate action: Create or select an immediate action if appropriate
```

Comment:
Immediate action is on Description tab, not Causes and Analysis.
![image](https://github.com/user-attachments/assets/04ce313c-6ce2-4272-b7d2-5aaa53aa5de5)